script.js

```diff
@@ -2,13 +2,11 @@ const btn = document.querySelector('#btn');
 const list = document.querySelector('#list');
 const form = document.querySelector('form');
 
-function bookObject(title, author) {
-  const books = {
-    title: title.value,
-    author: author.value,
-  };
-
-  return books;
+class BookObject {
+  constructor(title, author) {
+    this.title = title.value;
+    this.author = author.value;
+  }
 }
```

Comment:
- [ ] All your methods for adding, removing, and fetching the books should be within this class. Right now they are outside of it. Kindly refactor your code accordingly.